packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1329,8 +1402,23 @@ class RenderCustomPaint extends RenderProxyBox {
     RenderBox child
   }) : _painter = painter, _foregroundPainter = foregroundPainter, super(child);
 
+  /// The background custom paint delegate.
+  ///
+  /// This painter, if non-null, is invoked to paint behind the children.
```

Comment:
This class could be used by non-framework code, so i am reluctant to explicitly say the framework is the caller.

The children are really render objects here (this is the layer below the widgets). I'm not sure what to say exactly. Technically there's only one child for this class. I hesitate to say "descendants", since some might not paint (consider e.g. RenderOffStage). Hence my compromise on "children".